src/views/List.jsx

```diff
@@ -134,26 +150,36 @@ export function List({ data, listToken, loading, confirmLogOut }) {
 				</>
 			)}
 			<ul>
-				{/* filter through groups array for each group to display by time frame */}
-				{groups.map((group) => {
-					return (
-						<section className={group.timeFrame}>
-							<h1>{group.timeFrame}</h1>
-							{searchResults
-								// within each group's filteredData, map through to each item to pass in as a prop
-								.filter((item) => group.filteredData(item))
-								.map((filteredItem) => {
-									return (
-										<ListItem
-											key={filteredItem.id}
-											item={filteredItem}
-											listToken={listToken}
-										/>
-									);
-								})}
-						</section>
-					);
-				})}
+				<div className="group-container">
+					{/* filter through groups array for each group to display by time frame */}
+					{groups.map((group) => {
```

Comment:
I think if we add ```data.length === 0 ? <></> :``` here on line 155 right before groups.map, I think this should remove the group renderings when there are no items. Then, when an item is added, all groups should appear and the item will be in the appropriate group.

Reply:
Without items added:

![Screen Shot 2022-08-28 at 10 15 15 AM](https://user-images.githubusercontent.com/90880701/187086383-b31f5023-2d0f-4b20-845a-09978ba93081.png)

With an item added:
![Screen Shot 2022-08-28 at 10 16 03 AM](https://user-images.githubusercontent.com/90880701/187086417-c9acbef2-e8c8-4ab4-aa4b-601d81914bbe.png)

Reply:
I like that suggestion @Trevor-Rezac. 👏🏻